### PR TITLE
Added GeoIps to loadbalancers

### PR DESCRIPTION
**Pull request by Gman98ish**

### CODE REVIEW

File: src/Loadbalancers/ListenerClient.php [hunk 1]
```diff
@@ -202,6 +208,9 @@ public function deleteBindById($id, $bindId)
      */
     public function loadEntity($data)
     {
-        return new Listener($this->apiToFriendly($data, $this->getEntityMap()));
+        $listener = new Listener($this->apiToFriendly($data, self::MAP));
+        $listener->geoip = new GeoIp($this->apiToFriendly($data->geoip, self::GEOIP_MAP));
```
Comment (rbibby) [inline src/Loadbalancers/ListenerClient.php:212]: What happens if `geoip` is null in the API, will this work?

**Review by rbibby — COMMENTED** [review]
Left a question but other than that looks fine

**Review by rbibby — APPROVED** [review]

**Comment by phily245** [comment]
@Gman98ish @rbibby this is approved, does this need merging in?

**Comment by rbibby** [comment]
> @Gman98ish @rbibby this is approved, does this need merging in?

The API for it hasn't been released yet

**Review by bluemorbo — APPROVED** [review]
LGTM

**Pull request merged**